collision/space.go

```diff
@@ -128,20 +130,19 @@ func (s *Space) Overlap(other *Space) (xOver, yOver float64) {
 	return
 }
 
-// SetDim sets the dimensions of the space
-func (s *Space) SetDim(w, h float64) {
-	s.Update(s.GetX(), s.GetY(), w, h)
-}
-
 // OverlapVector returns Overlap as a vector
 func (s *Space) OverlapVector(other *Space) physics.Vector {
 	xover, yover := s.Overlap(other)
+	// Todo: why are we multiplying by -1 here, shouldn't that
+	// also be happening in Overlap at least?
```

Comment:
This is actually fairly weird... Overlap returns only negative number so that ensures that Overlap Vector is always positive. It would seem that the Overlap function returning negative is likely only relevant for legacy usages.